tests/nightly/test_large_vector.py

```diff
@@ -168,6 +168,12 @@ def test_topk():
     val = nd.topk(b, k=1, axis=0, dtype=np.int64, ret_typ="value")
     assert val.sum() == (LARGE_X - 1)
 
+    
+def test_mean():
+    a = nd.arange(-LARGE_X / 2, LARGE_X / 2 + 1, dtype=np.int64)
```

Comment:
nit: even though LARGE_X is divisible by 2 but can we use integer division here `LARGE_X//2`